Add github collaboration guide

nazreenpe opened the pull request:
Added a study link for github collaboration guide.

alodahl commented:
OOps, I didn't see that you had merged this - I've had the window open for a while.  @vegetabill what do you think about my review comment?

nazreenpe commented:
Hi Alina,
Thank you for pointing out the error. 
I fixed the error in the link. 
It was supposed to be an actual article on basics of github collaboration. I found in really useful for beginners. And it does not have advanced information, which is why I added in the study material section itself.
